From 0803b8d940b2bd91934b2411cdd90eb010411826 Mon Sep 17 00:00:00 2001
From: arthurlai <v-arthurlai@microsoft.com>
Date: Tue, 16 Mar 2021 15:52:15 +0800
Subject: [PATCH 1/2] Fix Hash for Release Package

---
 mk/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/common.mk b/mk/common.mk
index 859895b..c5fa6d8 100644
--- a/mk/common.mk
+++ b/mk/common.mk
@@ -22,7 +22,7 @@ PATCH    ?= patch
 UID      := $(shell id -u)
 GID      := $(shell id -g)
 DATE     := $(shell date -u --iso-8601=minutes)
-REVISION := $(shell git rev-parse HEAD)
+REVISION := bd9fc3f2b642345301cb2e23de07ec5386232317
 COMPILER := $(realpath $(shell which $(CC)))
 PLATFORM ?= $(shell uname -m)
 
-- 
2.17.1

